syntax = "proto3";

package bookstore;

import "order.proto";

service FraudDetectionService {
    rpc InitDetectFraud (OrderData) returns (EmptyMessage);
    rpc DetectUserFraud (OrderInfo) returns (OrderResponse);
    rpc DetectCreditCardFraud (OrderInfo) returns (OrderResponse);
    rpc ClearData (OrderInfo) returns (ClearDataResponse);
}
